include/deal.II/differentiation/ad/ad_number_traits.h

```diff
@@ -1009,6 +1009,62 @@ namespace Differentiation
         typename internal::RemoveComplexWrapper<ScalarType>::type,
         ADNumberTypeCode>::n_supported_derivative_levels;
 
+#  else
+
+    template <typename ScalarType, enum NumberTypes ADNumberTypeCode>
+    const bool NumberTraits<
```

Comment:
This has to be `constexpr` to be consistent with the declaration.

Reply:
Doh. Although incorrect, this happily compiled locally with GCC 6.3.0 :-/

Reply:
Thanks for the tip, btw.